src/components/search/concept-modal/tabs/cdes/cde-item.js

```diff
@@ -19,7 +23,57 @@ const Section = ({ title, children }) => (
     </Space>
 )
 
-export const CdeItem = ({ cde, cdeRelatedConcepts, highlight }) => {
+const RelatedStudiesList = ({relatedStudySource}) => {
+    const { analyticsEvents } = useAnalytics()
+    // const studyLinkClicked = () => {
+    //   analyticsEvents.studyLinkClicked(study.c_id)
+    // }
+    const [showMore, setShowMore] = useState(false)
+
+    if (!relatedStudySource) return (
+        <Text style={{ fontSize: 13, color: "rgba(0, 0, 0, 0.45)" }}>We couldn&apos;t load any related studies.</Text>
+    )
+
+    // Sort the related studies.
+    const relatedStudies = relatedStudySource.sort((a, b) => { return b.c_id < a.c_id});
+
+    if (relatedStudies.length === 0) return (
+        <Text style={{ fontSize: 13, color: "rgba(0, 0, 0, 0.45)" }}>We don&apos;t know of any related studies.</Text>
+    )
+
+    return (
+        <div>
+            <List
+                size="small"
+                dataSource={showMore ? relatedStudies : relatedStudies.slice(0, SHOW_MORE_CUTOFF)}
+                renderItem={(study) => (
+                    <List.Item key={study.c_id}>
+                    <List.Item.Meta
+                        size="small"
+                        description={
+                            <>
+                            <font size="-1"><a href={study.c_link} target="_blank" rel="noopener noreferrer">{study.c_id}</a></font>: {study.c_name}
+                            </>
+                        }
+                    />
+                    </List.Item>
+                )}
+            />
+            { relatedStudies.length >= SHOW_MORE_CUTOFF && (
```

Comment:
Think this should be >, not >=. Also, the content is triple indented for some reason.